packages/hash/integration/src/tests/model/knowledge/page.model.test.ts

```diff
@@ -39,10 +39,10 @@ describe("Page model class", () => {
 
   const createBlock = async () =>
     await BlockModel.createBlock(graphApi, {
-      accountId: testUser.accountId,
+      ownedById: testUser.ownedById,
```

Comment:
Should this be
```suggestion
      ownedById: testUser.entityId,
```
to not get the workspace account?